spring-cloud-gcp-data-firestore/src/test/java/org/springframework/cloud/gcp/data/firestore/it/FirestoreIntegrationTestsConfiguration.java

```diff
@@ -65,7 +65,7 @@
 	@Bean
 	public FirestoreTemplate firestoreTemplate(FirestoreGrpc.FirestoreStub firestoreStub,
 			FirestoreClassMapper classMapper) {
-		return new FirestoreTemplate(firestoreStub, this.defaultParent, classMapper);
+		return new FirestoreTemplate(firestoreStub, this.defaultParent, classMapper, new FirestoreMappingContext());
```

Comment:
`FirestoreMappingContext` bean already exists in the configuration class. Simply wire it into the method.